### ISSUE TITLE
Ficha del Alumno

### ISSUE THREAD

**Issue by Nikelondeon** [proyectoalba/alba2#13]
En observaciones generar un espacio destacado o un pop up que permita identificar rápidamente información sensible, por ejemplo que es dependiente de una medicación o alguna enfermedad que pueda resultar "complicada" para el tratamiento; o también si ese alumno debe retirarse del establecimiento con el padre y este tiene orden de alejamiento o similar. Pensando en casos de violencia familiar o separaciones conflictivas con restriccion.

**Comment by lucianobaraglia** [comment]
Está contemplada la "Ficha de Salud" principal, que es al momento y durante el proceso de la inscripción, y además tenemos lo que son "Actualizaciones de Salud", para ir en definitiva, actualizando los datos.
Cierra por ese lado, @Nikelondeon ?

**Comment by Nikelondeon** [comment]
Sí, a lo que me refiero es que se pueda hacer alguna observación como IMPORTANTE para que apenas se consulte el legajo del alumno salte a primera vista. Muchas veces la información "importante" se pierde en la burocracia de las planillas y cuando surge alguna emergencia no hay tiempo ni memoria para recurrir al lugar específico.

**Comment by lucianobaraglia** [comment]
Suena perfecto!
Te jode poner un ejemplo específico de qué se cargaría? Así podemos ver si cierra lo que tenemos modelado hasta ahora o hay que modificar.

Graciaaaas!! :smiley:

**Comment by Nikelondeon** [comment]
El alumno Carlitos Menem (jejeje), a la hora de salida se queda encerrado en el salón.
Contrario a la lógica, no quiere irse a su casa, como tampoco tiene intensión de acercarse a la puerta.
Su Preceptora no está (por el motivo que se te ocurra), por lo tanto la Preceptora del curso de al lado, si bien conoce a Carlitos, no lo hace en profundidad. Mientras contenido por otros docentes, se busca en ALBA el legajo de Carlitos. Apenas se hace click en el nombre surge un POPUP con la leyenda: "Los padres tuvieron un divorcio conflictivo, el padre tiene una orden de restricción hacia la familia." Obviamente, sin exponer el alumno a una posible situación peligrosa, se puede "hipotetizar" que el padre lo podría estar esperando en algún tramo del recorrido a su domicilio.
Se llama a la madre y en caso de ser necesario, a la fuerza pública para acompañar a la madre y a su hijo.

PARECE FICCIÓN, pero lamentablemente es un hecho verídico del cual debí participar. Salió todo bien, pero si hubiera habido una herramienta así, el Padre no se hubiera encontrado con el hijo.

**Comment by lucianobaraglia** [comment]
Ufffh...impactante...
Cómo lo llamarías dentro del sistema?
Por ejemplo, vas a la ficha del alumno y querés cargar uno de estos datos.
¿Cómo te imaginás el nombre en el menú para cargarlo? (eso nos va a dar una idea)

**Comment by Nikelondeon** [comment]
"Nota Destacada" o "Aviso Destacado"....
Me mataste con el nombre, no lo había pensado.

**Comment by lucianobaraglia** [comment]
Si te parece lo podemos llamar **Información Sensible**, y al acceder a la ficha del alumno se vería en forma destacada.

**Comment by Nikelondeon** [comment]
Perfecto!

**Comment by lucianobaraglia** [comment]
Me surge una duda con respecto a los alumnos...
Ponéle que el establecimiento dicta más de un nivel, supongamos primaria y secundaria.
Cuando el alumno pasa a otro nivel, se crea un nuevo legajo/historial/ficha de salud/historial de conducta/etc, etc?

En definitiva, ¿qué datos del alumno quedan activos cuando pasa de nivel?

**Comment by lucianobaraglia** [comment]
También se me ocurre que tal vez inicien una nueva ficha por año/ciclo lectivo.
Entonces esta misma pregunta (¿se crea una ficha nueva por año?) te la hago con respecto a estos datos:
- Ficha de datos del alumno
- Ficha de salud
- Historial de conducta

Y alguna otra se me va a ocurrir.

**Comment by Nikelondeon** [comment]
El legajo del alumno no debería caducar nunca. Lo más interesante de la digitalización es acceder de manera sencilla a toda la información, tanto la histórica como la reciente.

Respondiendo a tu pregunta, NO. No se crea un legajo nuevo, se actualiza (en caso de ser necesario). Sería bueno que se pueda ver, en algún rincón, fecha de última actualización.

Ya sea en el mismo Establecimiento o en otro, poder tener el legajo con el historial (conducta, calificaciones, incidencias, etc.) es una herramienta fabulosa.

Por ejemplo: Un alumno va de la Escuela A a la Escuela B; al hacer el "pase" (estado administrativo obligatorio) que se imprima todo el legajo con el historial incluído debería ser una posibilidad.
Entonces cuando el alumno llega a la Escuela B, además de la "constancia de pase" hace entrega de su Legajo. Con esto la Escuela B no tiene que empezar de nuevo para acompañar el desarrollo del alumno.

**Comment by lucianobaraglia** [comment]
Excelente!!!
Y con las fichas de salud y el historial de conducta? No se empieza uno por año y se "archiva" el anterior?

**Comment by hectorsanchez** [comment]
Acordate que en ALBA 1.0 tenemos algo que se llama "Legajo pedagogico" , creo que no se debería almacenar en el sistema información sensible del alumno.

**Comment by lucianobaraglia** [comment]
@hectorsanchez el legajo sería la unión de varias de estas fichas (salud, conducta, datos personales).
Y lo de almacenar información sensible ya es una decisión que creo que no podemos manejar...eso depende del establecimiento.

**Comment by Nikelondeon** [comment]
@hectorsanchez No entiendo por qué "no debería almacenar el sistema" información (llegado el caso, todos los datos son sensibles). Justamente, digitalizar el legajo del alumno y toda su información es una herramienta muy interesante. No sería muy criterioso que sí exista un legajo en papel con información sensible y el legajo digital este incompleto, estamos desestimando la herramienta.
Quizás no entendí tu preocupación y te contesté cualquier cosa, pero un ejemplo del riesgo supuesto sería de ayuda.

@lucianobaraglia La información que se archiva es la que deja de ser dinámica (que puede variar con el tiempo), y en todo caso, poder acceder para consulta la evolución del alumno en su paso por el Establecimiento es muy útil desde lo pedagógico y lo administrativo.
Alguna enfermedad, diagnóstico o sanción disciplinar modificará las fichas respectivas del legajo.
